app/views/dashboard/show.html.erb

```diff
@@ -9,7 +9,11 @@
         </h2>
         <p>Choose a delivery partner and upload schools using a CSV.</p>
         <h2 class="govuk-heading-m">
-          <%= govuk_link_to "Check your schools for #{Cohort.current.display_name}", lead_providers_your_schools_path %>
+          <% if FeatureFlag.active?(:multiple_cohorts) %>
+            <%= govuk_link_to "Check your schools", lead_providers_your_schools_path(cohort: Cohort.next) %>
```

Comment:
If this is just about the copy change I'd drop this condition entirely and just change it to "Check your schools" immediately.

Reply:
I agree, that should be fine. I've pushed the changes.